java/flight/flight-core/src/main/java/org/apache/arrow/flight/grpc/RequestContextAdapter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Does there need to be a date?

Reply:
Consistent with headers used for this codebase.